easy_rec/python/utils/config_util.py

```diff
@@ -24,6 +24,21 @@
   tf = tf.compat.v1
 
 
+def search_pipeline_config(directory):
+  dir_list = []
+  for root, dirs, files in tf.gfile.Walk(directory):
+    for f in files:
+      _, ext = os.path.splitext(f)
+      if ext == '.config':
+        dir_list.append(root)
+  if len(dir_list) == 0:
+    raise ValueError('config is not found in directory %s' % directory)
+  elif len(dir_list) > 1:
+    raise ValueError('config saved model found in directory %s' % directory)
+
+  return dir_list[0]
```

Comment:
return a directory? should be a pipeline.config

Reply:
return pipeline.config , add test in predictor_test.py PredictorTestOnDS.test_local_pred_without_config